docs/operations/mediated_devices_configuration.md

```diff
@@ -0,0 +1,121 @@
+# Automatic configuration of mediated devices / vGPUs
+
+KubeVirt aims to facilitate the configuration of mediated devices on large clusters.
+The new `mediatedDevicesConfiguration` API in KubeVirt CR allows adimnistrators to
+create or remove mediated devices in a declative way, by providing a list of the desired mediated devices types  that they expect to be configured in the cluster.
+
+It would also be possible to provide a more specific configuration that targets one or a group of nodes directly.
+
+KubeVirt will use the provided configuration to automatically create the relevant mdev/vGPU devices on nodes that can support it.
+
+The maximum amount of instances of the selected mdev type will be configured per card.
+
+> Some vendors, such as NVIDIA, require a driver to be installed on the nodes to provide mediated devices, including vGPUs.
+
+##### Here's an example of a configuration that can be provided to KubeVirt CR:
+```
+spec:  
+  configuration:  
+     mediatedDevicesConfiguration:  
+       mediatedDevicesTypes:  
+       - nvidia-222  
+       - nvidia-228  
+       nodeMediatedDevicesTypes:  
+       - nodeSelector:  
+         kubernetes.io/hostname: nodeName  
+         mediatedDevicesTypes:  
+         - nvidia-234
+```
+
+## Configuration scenarios
+### Large cluster with multiple cards on each node
+
+On nodes with multiple cards that can support similar vGPU types, the relevant desired types will be created in a round-robin manner.  
+
+#### For example, considering the following configuration:  
+```
+mediatedDevicesConfiguration:
+  mediatedDevicesTypes:  
+  - nvidia-222  
+  - nvidia-228
+  - nvidia-105
+  - nvidia-108
+```
+* Nodes with 2 Tesla T4 cards
+  * Each card can support multiple types: 
```

Comment:
I think we also mention,
Card does support multiple types, but only 1 type, is allowed "at the same time".

I know, this is something "according to the vendor directions", but was  thinking we still explicitly mention it.